pymobiledevice3/tcp_forwarder.py

```diff
@@ -87,22 +94,37 @@ def _handle_close_or_error(self, from_sock):
         self.logger.info(f'connection {other_sock} was closed')
 
     def _handle_data(self, from_sock, closed_sockets):
-        data = from_sock.recv(1024)
-
+        self.logger.debug("Handling data from %s", from_sock)
+        try:
+            data = from_sock.recv(1024)
+        except OSError:
```

Comment:
why this should be any different handling than `ConnectionResetError`? That way we can avoid code duplication

Reply:
Since BrokenPipe and other posix related errno's that can occur with no reason

Reply:
let's instead initialize `data = None` before the `try` statement, and change the `if len(data)` to just `if data` to avoid the duplication

Reply:
`if data` is an antipattern IMO. I will change it if you ask for it